src/_css/index.css

```diff
@@ -21,4 +21,18 @@
   .indent {
     text-indent: 1.5em;
   }
+
+  .hover\:bg-white-opac-10:hover {
+    background-color: rgba(255, 255, 255, 0.1);
+  }
```

Comment:
This class is unnecessary, we can replace it with `hover:bg-white hover:opacity-10`